packages/cli/src/index.ts

```diff
@@ -340,6 +346,11 @@ yargs
             type: 'boolean',
             default: false,
           },
+          'ignore-deployment-failures': {
```

Comment:
Suggest that we might rename this option as above

Reply:
I have renamed this parameter to `continue-on-deployment-failures`